charts/minecraft/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: minecraft
-version: 2.0.13
+version: 2.0.14
```

Comment:
As this is adding a new feature, can you bump the chart minor version?  In this case, it looks like it would be `2.1.0`.